src/script/editor/operations/op.js

```diff
@@ -114,6 +87,9 @@ function AtomDelete(aid) {
 		restruct.atoms.delete(this.data.aid);
 		restruct.markItemRemoved();
 
+		// TODO: - here
```

Comment:
Investigate this moment please. Check when in called and what can be missed on that step

Reply:
Deleted it